opensrp-reveal/src/main/java/org/smartregister/reveal/presenter/EventRegisterFragmentPresenter.java

```diff
@@ -92,6 +94,21 @@ private String mainSelect(String tableName, String mainCondition) {
     }
 
     protected String[] mainColumns(String tableName) {
+        if(Country.KENYA.equals(BuildConfig.BUILD_COUNTRY)){
```

Comment:
This is code repetition. The only difference here is the new `DATA_COLLECTION_DATE`. Why not just optionally add it to the column array when the build country is KENYA?

Reply:
yes you're right,let me update that.

Reply:
I've addressed this.